fisher.fish

```diff
@@ -9,6 +9,7 @@ function fisher -a cmd -d "fish package manager"
     set -g fisher_config $XDG_CONFIG_HOME/fisher
 
     set -q fisher_path; or set -g fisher_path $fish_config
+    set -l fishfile "$fish_config/fishfile"
```

Comment:
@mattmc3 If I get this right, `fishfile` is defined only once and never changes again. If so, this should be `-g`.

Reply:
Solid feedback! I pushed a new commit, which I'm real happy with how it simplifies the whole feature. One global, used everywhere `$fishfile` needs referenced.